documentation/book/assembly-encryption.adoc

```diff
@@ -0,0 +1,34 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context: {ContextProduct}
+
+include::common/attributes.adoc[]
+
+[id='encryption-{context}']
+= Encryption
+
+The Strimzi project supports data encryption of communication between the different Kafka and Strimzi components (Kafka brokers, Zookeeper nodes, Kafka Connect and Strimzi Topic Operator) through the SSL/TLS protocol.
```

Comment:
Strimzi (3x) should be the attribute

Reply:
Change this sentence to: 

Strimzi encrypts communication between the Kafka and Strimzi components through the SSL/TLS protocol.